Updated the cloud portal article with the new dashboard (#4417)
umbraco/UmbracoDocs: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md

When you log in to the [Umbraco Cloud Portal](https://www.s1.umbraco.io/home/login/) you will be presented with a view of all the Umbraco Cloud projects you've created or have been added to as a team member.

![View all projects](images/Umbraco-cloud-dashboard.png)

> On the project dashboard, you get an overview of all your projects, the environments, the usage for each project, which plans it is on, and whether it is a baseline or baseline-child project.

Author: ```suggestion
The projects dashboard gives an overview of all your projects, their environments, usage for each project, which plans it's on, and whether it's a baseline or baseline-child project.
```

> On the project dashboard, you get an overview of all your projects, the environments added, the usage for each of the project, which plan it is on, and whether it is a baseline or baseline-child project.

Author: ```suggestion
On the project dashboard, you get an overview of all your projects, the environments, the usage for each project, which plans it is on, and whether it is a baseline or baseline-child project.
```

In the top-right corner of the Cloud Portal, you will find:

- **Create New Project** - Allows you to create more projects using the plan you wish and a project will be ready for you within a few minutes.

- **Notifications** - You can also see notifications for your different projects. For example: if your project has been automatically updated or if an upgrade has failed.

- **Profile** - Manage projects, subscriptions, pending invites, organization information, profile details, view release notes, and log out of the portal.

### Project Groups

To give a better overview over your projects it is possible to order your projects into **Groups**.

This can be done by clicking the **Edit Group** button in the top left side.

![Edit Groups](images/edit-group.png)

> When clicking on **Edit Groups** you can create new Groups to sort your project in and create a better overview for yourself.

Reviewer: ```suggestion
After clicking on **Edit Groups**, you can create new Groups to sort your project in and create a better overview for yourself.
```

![Create Group](images/add-group.png)

> Once you click **Add Group** you can give it a name that fit the group, and then you can drag and drop your projects into the group of your choice.

Reviewer: ```suggestion
Click **Add Group** to give it a name and then drag and drop your projects into the group of your choice.
```
